src/editor.c

```diff
@@ -1234,15 +1236,65 @@ editor_get_indent_prefs(GeanyEditor *editor)
 }
 
 
+static inline gint compare_line_indentation(ScintillaObject *sci, gint line1, gint line2)
```

Comment:
do we use inline everywhere now? :)
Not that I mind, really, just that it must be the only few places where it's used, and if it does anything (does it?), there probably are other places we should start using it.

Also I'm always doubtful of the actual gain of using it.  Optimizing compilers (all of the current ones, probably) won't have a problem inlining a function not tagged `inline` if they see benefit, so then telling them they should do it means that we
* know what we're doing, and that inlining *is* better in this case (and remains until we remove the tag);
* think the compiler won't figure this out by itself, let alone know better.

My naive guess is that in any non-critical situation, the compiler will know better, because it will actually ask itself whether it should inline or not this particular function.  And if it's only used once, it probably will inline it anyway.